package.json

```
@@ -54,12 +54,12 @@
     "motion": "^12.23.24",
     "react": "^19.0.1",
     "react-dom": "^19.0.1",
-    "vite": "^6.2.3"
+    "vite": "^6.2.3",
+    "tailwindcss": "^4.1.14"
   },
   "devDependencies": {
     "@types/express": "^4.17.21",
     "@types/jszip": "^3.4.1",
```

Biome (2.5.7)

[error] The key `@types/jszip` was already declared.

(lint/suspicious/noDuplicateObjectKeys)